charts/docker-hub-rate-limit-exporter-chart/values.yaml

```diff
@@ -3,16 +3,26 @@
 # Declare variables to be passed into your templates.
 
 config:
-    exporterPort: 80 ## if you want to change the port exposed by the docker-hub-rate-limit-exporter
-    exporterVerbosity: 1 ## the loglevel
-    dockerhubUsername: ## put your dockerhub username here
-    dockerhubPassword: ## put your dockerhub password here
+    verbosity: 1 ## the loglevel
```

Comment:
I like this change. Here I would accept a lag of backward compatibility for the gain of readability since it would not break the application if one misses it.

Reply:
nice :smile: